.github/workflows/ci-fork.yml

```
@@ -0,0 +1,43 @@
+name: CI (Fork)
+
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types:
+      - completed
+
+jobs:
+  unit-test-results:
+    name: Unit Test Results
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.conclusion != 'skipped' &&
+      github.event.workflow_run.head_repository.full_name != github.repository
+
+    steps:
+      # we are not using actions/download-artifact here as
+      # it does not support downloading from a different workflow
+      # https://github.com/actions/download-artifact/issues/3
+      - name: Download Artifacts
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          artifact_url=${{ github.event.workflow_run.artifacts_url }}
+          artifact_url=$(gh api $artifact_url -q '.artifacts[] | select(.name=="unit-test-results") .archive_download_url')
```

I think we should either name the URL to list artifacts `artifacts_url` (first suggestion), or simply fold it (second suggestion).

```suggestion
          artifacts_url=${{ github.event.workflow_run.artifacts_url }}
          artifact_url=$(gh api $artifacts_url -q '.artifacts[] | select(.name=="unit-test-results") .archive_download_url')
```

```suggestion
          artifact_url=$(gh api ${{ github.event.workflow_run.artifacts_url }} -q '.artifacts[] | select(.name=="unit-test-results") .archive_download_url')
```

---

I like to keep the lines readable hence separate, and not combine all the logic into a one-liner